CalcLength multiply method

njlawton opened the pull request:
Implemented method to multiply CalcLength objects by a multiple

Commit:
Started implementing multiply method for calc-length
Created method outline in calc-length.js and created simple unit test for unimplemented method

Commit:
Implemented multiply method in calc-length.js
Implemented method passes simple unit test further tests are required to ensure its correctness

Commit:
Added test case for multiplication of calc-length
Added test case for decimal multiple to ensure correct behaviour for no interger multiples

wilddamon commented on src/calc-length.js:
```diff
@@ -64,6 +64,21 @@
+    var calcDictionary = new Object;
+
+    //iterate through all length types and multiply all non null lengths 
+    for(var index in shared.LengthValue.LengthType){
```
It's probably less confusing (and more common) to write

for (var i = 0; i < shared.LengthValue.LengthType.length; i++) {
var type = shared.LengthValue.LengthType[i];
...etc...
}

wilddamon commented on src/calc-length.js:
```diff
@@ -64,6 +64,21 @@
 
+  // Length Calculation Methods
+  CalcLength.prototype.multiply = function(multiplier) {
+    var calcDictionary = new Object;
```
Can just write

var calcDictionary = {};

Commit:
Changed multiplication method implementation in CalcLength class
Changed method of iteration to a standard for loop for improved code style compliance

Commit:
Improved multiplication method in CalcLength class
Changed the way calcDictionary is created into a way more consistant with code style used at google
